client/components/Cart.jsx

```diff
@@ -8,42 +8,40 @@ const propTypes = {
     productId: PropTypes.number.isRequired,
   })).isRequired,
   products: PropTypes.arrayOf(PropTypes.shape({
-    productId: PropTypes.number.isRequired,
-    name: PropTypes.number.isRequired,
-    price: PropTypes.number.isRequired,
+    id: PropTypes.number.isRequired,
+    name: PropTypes.string.isRequired,
+    price: PropTypes.string.isRequired,
```

Comment:
price should be a number

Reply:
Yes, price should be a number but its cominh through as a string, thats why I changed it